### ISSUE TITLE
Tabs: Add page navigation variant

### ISSUE THREAD

**Issue by majornista** [adobe/react-spectrum#4139]
# 🙋 Feature Request

Product teams are tempted to use Tabs to achieve the visual style and keyboard interaction behavior of Tabs for site navigation. But for this use case, each Tab would need to be a Link with an `href`, using `aria-current="page"` to specify the currently selected Link.

## 🤔 Expected Behavior

It would great if React-Spectrum had a semantic and accessible ready-to-use component for page navigation. The component would have the visual styling of Tabs, but would have the semantics of a list of Links, and like Tabs would behave as a Picker in responsive mode on mobile devices.

## 😯 Current Behavior

A product team uses Tabs with Items that contain a Link. This may allow Links be opened in a new browser Tab from a context menu, but it creates an accessibility failure due to nested interactive elements, and keyboard navigation problems using Tab key and Shift+Tab keys.

## 💁 Possible Solution

Create a "navigation" variant of Tabs that is styled the same but uses a Link instead of a Tab for each of the Items.

## 🧢 Your Company/Team

Adobe/Accessibility, Adobe/DCES
 
## 🎁 Tracking Ticket (optional)

DCES-4421687 and DCES-4418118

**Comment by devongovett** [comment]
I think this would be a separate component from Tabs. It wouldn't have any of the behaviors of tabs and follows a different accessibility pattern. Not sure there'd even be much JavaScript needed for this since they are just links, so teams could easily reuse Spectrum CSS for now.

Related to this is the On Page Navigation component in Quarry, which also looks like tabs but behaves more like side navigation.

**Comment by snowystinger** [comment]
Closing as nothing to do here and will be easier in s2
